src/index.spec.ts

```diff
@@ -1,777 +1,898 @@
+            expect(outputPath).to.equal(path.join(path.resolve(options.outDir), '.roku-deploy-staging'));
+        });
     });
-});
 
-describe('createPackage', function () {
+    describe('getOutputPkgFilePath', function () {
+        it('should return correct path if given basename', async () => {
+            options.outFile = 'roku-deploy';
+            let outputPath = rokuDeploy.getOutputPkgFilePath(options);
+            expect(outputPath).to.equal(path.join(path.resolve(options.outDir), options.outFile + '.pkg'));
+        });
 
-    it('should throw error when no files were found to copy', async () => {
-        try {
-            options.files = [];
-            await createPackage(options);
-            assert.fail('Exception should have been thrown');
-        } catch (e) {
-            assert.ok('Exception was thrown as expected');
-        }
+        it('should return correct path if given outFile option ending in .zip', async () => {
+            options.outFile = 'roku-deploy.zip';
+            let outputPath = rokuDeploy.getOutputPkgFilePath(options);
+            expect(outputPath).to.equal(path.join(path.resolve(options.outDir), 'roku-deploy.pkg'));
+        });
     });
 
-    it('should create package in proper directory', async function () {
-        await createPackage(options);
-        expect(file(getOutputFilePath())).to.exist;
+    describe('getOutputZipFilePath', function () {
+        it('should return correct path if given basename', async () => {
+            options.outFile = 'roku-deploy';
+            let outputPath = rokuDeploy.getOutputZipFilePath(options);
+            expect(outputPath).to.equal(path.join(path.resolve(options.outDir), options.outFile + '.zip'));
+        });
+
+        it('should return correct path if given outFile option ending in .zip', async () => {
+            options.outFile = 'roku-deploy.zip';
+            let outputPath = rokuDeploy.getOutputZipFilePath(options);
+            expect(outputPath).to.equal(path.join(path.resolve(options.outDir), 'roku-deploy.zip'));
+        });
     });
 
-    it('should only include the specified files', async () => {
-        try {
-            options.files = ['manifest'];
-            await createPackage(options);
-            let zip = new AdmZip(getOutputFilePath());
+    describe('createPackage', function () {
+        it('should throw error when no files were found to copy', async () => {
+            try {
+                options.files = [];
+                await rokuDeploy.createPackage(options);
+                assert.fail('Exception should have been thrown');
+            } catch (e) {
+                assert.ok('Exception was thrown as expected');
+            }
+        });
+
+        it('should create package in proper directory', async function () {
+            await rokuDeploy.createPackage(options);
+            expect(file(await rokuDeploy.getOutputZipFilePath(options))).to.exist;
```

Comment:
`getOutputZipFilePath` is not async, so remove the await

Reply:
Still crazy to me that's allowed. I guess it's up to PRs to uncover this sort of stuff :)